prototype.js issue (#5627)
segmentio/segment-docs: src/connections/sources/catalog/libraries/website/javascript/troubleshooting.md

The AJS cookies being set under segment.com are first-party cookies. They are part of Segment's own implementation as well as the destination Segment uses. These cookies are not related to your implementation of Segment, and you only see them because you've visited Segment's domain using the same browser. They are sent to the writekey connected to Segment's own workspace, and are associated with the events Segment tracks when you visit segment.com.

> ### Are you loading Prototype.js?

Reviewer: ```suggestion
### Loading with Prototype.js
```

> If you are having issues with your destinations loading, there is a [known issue that was reported](https://github.com/prototypejs/prototype/issues/338){:target="_blank"}.  
> 
> In order to prevent the issues, you can preserve the original `array.from` method without letting the prototype override it.

Reviewer: ```suggestion
If you're having issues with your destinations loading with Protoype.js, there is a [known issue that was reported](https://github.com/prototypejs/prototype/issues/338){:target="_blank"} regarding this.  

In order to prevent the issues, you can preserve the original `Array.from` method without letting the prototype override it.
```

> In order to prevent this from happening, you can preserve the original Array.from method, without letting the prototype override it.

Reviewer: ```suggestion
In order to prevent this from happening, you can preserve the original `array.from` method without letting the prototype override it.
```
